pkg/webui/console/store/reducers/devices.js

```diff
@@ -66,10 +67,11 @@ const devices = (state = defaultState, { type, payload, event }) => {
       return {
         ...state,
         selectedDevice: combineDeviceIds(payload.appId, payload.deviceId),
+        version_ids: payload.version_ids,
       }
     case UPDATE_DEV_SUCCESS:
     case GET_DEV_SUCCESS:
-      const updatedState = { ...state }
+      const updatedState = { ...state, version_ids: payload.version_ids }
```

Comment:
I don't get these changes. The `version_ids` should be part of the device entity stored in the store.